anchore_engine/services/policy_engine/engine/vulns/mappers.py

```diff
@@ -383,6 +400,9 @@ def to_engine(
     "centos": DistroMapper(
         engine_distro="centos", grype_os="centos", grype_like_os="fedora"
     ),
+    "busybox": DistroMapper(
+        engine_distro="busybox", grype_os="busybox", grype_like_os=""
```

Comment:
Double-checking, is the empty string for `grype_like_os` here and on line 432 intentional?